### PR TITLE
CreateTOWSON.json

### PR DESCRIPTION
**Pull request by pvolkaji1**
towson pin jason file

### CODE REVIEW

File: _pins/TOWSON.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: pvolkaji1
+latitude: 39.4015° N
```
Comment (hollenberry) [inline _pins/TOWSON.json]: Hey @pvolkaji1 ! It looks like the ° N and ° W are additional characters that our build isn't prepared to handle. You should be good to go if you remove these.

I also suggest double checking your coordinates on http://mynasadata.larc.nasa.gov/latitudelongitude-finder/
and plugging in those numbers if they're different. 😄

**Comment by pvolkaji1** [comment]
commit

**Comment by hollenberry** [comment]
Almost @pvolkaji1 ! Now we just need to remove the degree symbols.

**Comment by pvolkaji1** [comment]
Thanks @hollenberry i removed it.

**Comment by hollenberry** [comment]
Last item is the longitude. On the website above, it looks like it should be preceded by a ( - ) symbol.

**Comment by hollenberry** [comment]
Awesome! This looks good to 🚢 ! Go ahead once the build passes 😄 👍

**Comment by pvolkaji1** [comment]
@hollenberry Thank you for your help! i really appreciate you quick help. i've successfully merged the branch.

**Comment by hollenberry** [comment]
No problem! Happy to help! Feel free to delete your branch whenever you want to 😄

**Pull request merged**